app/javascript/controllers/form_input_controller.js

```diff
@@ -1,20 +1,28 @@
-import { Controller } from "@hotwired/stimulus";
+import { Controller } from "@hotwired/stimulus"
 
-// Connects to data-controller="request-item"
+// Connects to data-controller="form-input"
 export default class extends Controller {
-  static targets = ["addButton", "addDest", "addTemplate"];
+  static targets = ["addButton", "addTemplate", "removeContainer", "addContainer"];
 
-  addItem() {
-    const template = this.addTemplateTarget.content.firstElementChild.innerHTML;
+  // requires data-add-dest-selector to be set on the add button OR
+  // data-form-input-target="addContainer" to be set on the add container
+  addItem(event) {
+    const template = this.addTemplateTarget.content.firstElementChild.outerHTML;
+    const dest = document.querySelector(event.target.dataset.addDestSelector) ||
```

Comment:
If we're using Stimulus, we should be able to just pass this value directly into the controller via `value` data attributes: https://stimulus.hotwired.dev/reference/values

Reply:
Hmm... fair enough. I feel like there's a better way to accomplish this, but I can't think of one right now. :)